tests/suites/deploy/deploy_revision.sh

```diff
@@ -83,7 +83,7 @@ run_deploy_revision_refresh() {
 
 	# revision 21 is in channel latest/edge
 	wait_for "juju-qa-test" "$(charm_rev "juju-qa-test" 21)"
-	wait_for "juju-qa-test" "$(charm_channel "juju-qa-test" "latest/edge")"
+	wait_for "juju-qa-test" "$(charm_channel "juju-qa-test" "edge")"
```

Comment:
This shouldn't have changed - we need to understand this

Reply:
Seems to be a bug in both 2.9 and 3.0 where the track is not being filled in. Fixing it is probably beyond the scope of this PR

Reply:
verified, this change in #14470 hasn't been merged forward correctly.  investigating.